cluster/terraform_kubernetes/config/prometheus/datasources.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: 1
+datasources:
+- name: Thanos
+  type: prometheus
+  access: proxy
+  url: thanos-store-gateway.monitoring.svc.cluster.local
```

Comment:
this should be http://thanos-querier.monitoring.svc.cluster.local:9090

once updated, please confirm you can create a dashboard and see data (the dashboard doesn't need to be added to this PR, we set those up in a later PR)